From 380b8fc6ef4bc62831a48351191720bc5e12a72c Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sat, 04 Sep 2021 22:32:03 +0300
Subject: [PATCH] [SW-0] crt0: use tabs for alignment of parameters

Some assembly commands' parameters were aligned using spaces.
We should use tabs for aligning those.
Fix the white space.
No functional changes.

Change-Id: I2251ccaea31ca3723f60868f93d97582b56d29d5
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/app/ppboot/crt0_ppboot.S b/app/ppboot/crt0_ppboot.S
index 66f144b..7b6966e 100644
--- a/app/ppboot/crt0_ppboot.S
+++ b/app/ppboot/crt0_ppboot.S
@@ -18,11 +18,11 @@
 lf2flt:
 
 	/* Clear BSS */
-	mov     r5, __bss_start
-	sub     r6, __bss_end, r5
-	lsr.f   lp_count, r6, 2
-	lpnz    1f
-	st.ab   0, [r5, 4]
+	mov	r5, __bss_start
+	sub	r6, __bss_end, r5
+	lsr.f	lp_count, r6, 2
+	lpnz	1f
+	st.ab	0, [r5, 4]
 1:
 	/* copy data and sdata sections */
 	mov_s	r1, @_erodata		; r1 = end of the text/ro section
diff --git a/app/preboot/crt0_preboot.S b/app/preboot/crt0_preboot.S
index b329174..8c038b1 100644
--- a/app/preboot/crt0_preboot.S
+++ b/app/preboot/crt0_preboot.S
@@ -25,7 +25,7 @@
 	sub	r6, __stack_top, r5
 	lsr.f	lp_count, r6, 2
 	lpnz	1f
-	st.ab   0, [r5, 4]
+	st.ab	0, [r5, 4]
 1:
 #endif
 
